misc/formatter/modules/formatter-core/src/test/resources/declarations/class-definition/source/class_definition_declaration_2.bal

```diff
@@ -0,0 +1,17 @@
+class Person {
+    *A;
+    public int age = 10;
+    function foo() returns int {
+        int a = 12345;
+        return a;
+    }
+
+    public string name = "sample name";
+    int year = 50;
+    function bar() returns string {
+        string b = "bar";
+        return b;
+    }
+
+    string month = "february";
+}
```

Comment:
Shouldn't we disorganize the formatting here?

Reply:
Disorganized the formatting.